src/api/app/services/notified_projects.rb

```diff
@@ -0,0 +1,26 @@
+class NotifiedProjects
+  def initialize(notification)
+    @notification = notification
+    @notifiable = @notification.notifiable
+  end
+
+  def call
+    return [] unless @notifiable
+
+    case @notification.notifiable_type
+    when 'BsRequest'
+      @notifiable.target_project_objects.uniq
+    when 'Comment'
+      case @notifiable.commentable_type
+      when 'Project'
+        @notifiable.commentable
+      when 'Package'
+        @notifiable.commentable.project
+      when 'BsRequest'
+        @notifiable.commentable.target_project_objects.uniq
```

Comment:
if `target_project_objects` is a relation, wouldn't be better to use `distinct` here?

Reply:
True. Changed.